src/Symfony/Component/Serializer/Mapping/Loader/YamlFileLoader.php

```diff
@@ -91,6 +91,14 @@ public function loadClassMetadata(ClassMetadataInterface $classMetadata): bool
                     $attributeMetadata->setSerializedName($data['serialized_name']);
                 }
 
+                if (isset($data['serialized_path'])) {
+                    if (!$data['serialized_path']) {
```

Comment:
I think that we might ensure that `$data['serialized_path']` is an array, and that each element is a string as well.

```php
if (!\is_array($data['serialized_path'])) {
    throw ...
}
foreach ($data['serialized_path'] as $serilalizedPath) {
    if (!\is_string($serilalizedPath)) {
        throw ...
    }
}

Reply:
this will be caught later on I guess, let's keep the code simple here